cristin-biobank/build.gradle

```diff
@@ -18,28 +15,8 @@ dependencies {
 }
 
 
-tasks.named('test') {
+tasks.named('test', Test) {
     useJUnitPlatform {
         excludeTags 'karateTest'
     }
-    failFast = true
-    testLogging {
-        events 'skipped', 'passed', 'failed'
-    }
-    finalizedBy jacocoTestReport
```

Comment:
what does these do and why dont we need them?

Reply:
moved to `nva.cristin.service.core.java-conventions.gradle`

```
tasks.named('test', Test) {
    useJUnitPlatform()
    testLogging {
        events 'skipped', 'passed', 'failed'
    }
```